runtime/sema/type.go

```diff
@@ -1907,6 +1907,16 @@ It does not modify the original array.
 If either of the parameters are out of the bounds of the array, or the indices are invalid (` + "`from > upTo`" + `), then the function will fail.
 `
 
+var insertableEntitledAccess = NewEntitlementSetAccess(
+	[]*EntitlementType{InsertableEntitlement, MutableEntitlement},
+	Disjunction,
+)
+
+var removableEntitledAccess = NewEntitlementSetAccess(
+	[]*EntitlementType{RemovableEntitlement, MutableEntitlement},
+	Disjunction,
+)
```

Comment:
These entitlement mappings are re-used across functions.

@dsainati1 Wanted to double check whether it is safe to re-use these, and to ensure these mappings are not mutated in the downstream codes.